sevntu-checkstyle-sonar-plugin/src/main/resources/com/github/sevntu/checkstyle/sonar/checkstyle-extensions.xml

```diff
@@ -921,6 +921,31 @@
     <configKey>Checker/TreeWalker/com.github.sevntu.checkstyle.checks.coding.RequireFailForTryCatchInJunitCheck</configKey>
   </rule>
 
+  <rule>
+    <key>com.github.sevntu.checkstyle.checks.coding.Jsr305AnnotationsCheck</key>
+    <name>Line Length Extended</name>
+    <category name="coding"/>
+    <description>Checks for long lines.</description>
+    <configKey>Checker/TreeWalker/com.github.sevntu.checkstyle.checks.coding.Jsr305AnnotationsCheck</configKey>
+
+    <param key="packages" type="STRING">
+      <defaultValue></defaultValue>
+      <description>Packages to be checked.</description>
+    </param>
+    <param key="excludePackages" type="STRING">
+      <defaultValue></defaultValue>
+      <description>Packages excluded from checking.</description>
+    </param>
+    <param key="allowOverridingReturnValue" type="BOOLEAN">
+      <defaultValue>false</defaultValue>
+      <description>Allow Overriding return values, useful for upgrading.</description>
+    </param>
+    <param key="allowOverridingParameter" type="BOOLEAN">
+      <defaultValue>false</defaultValue>
+      <description>Allow Overriding paramaters, useful for upgrading.</description>
```

Comment:
`paramaters` => `parameters`

Reply:
Done.

Reply:
@mbert not done.

Reply:
Of course, the same typo was in the xml file, too. I'll look into this.